papers/bradley_dice/55_bradley_dice.rst

```diff
@@ -0,0 +1,552 @@
+We will discuss recent improvements to the software's feature set, scalability, scientific applications, usability, and community.
+Newly implemented synced data structures, features for generalized workflow execution, and performance optimizations will be covered, as well as recent research using the framework and the project's efforts on improving documentation, contributor onboarding, and governance.
+
+.. class:: keywords
+
+   data management, data science, database, simulation, collaboration, workflow, HPC, reproducibility
+
+
+Introduction
+------------
+
+.. figure:: signac_overview.pdf
+    :align: center
+    :scale: 40 %
+    :figclass: w
+
+    Overview of the **signac** framework.
+    Users first create a project, which initializes a workspace directory on disk.
+    Users define state points which are dictionaries that uniquely identify a job.
+    The workspace holds a directory for each job, containing JSON files that store the state point and job document.
+    The job directory name is a hash of the state point's contents.
+    Here, the ``init.py`` file initializes an empty project and adds one job with state point ``{"a": 1}``.
+    Next, users define a workflow using a subclass of **signac-flow**'s ``FlowProject``.
+    The workflow shown has three operations (simulate, analyze, visualize) that, when executed, produce two new files ``results.txt`` and ``plot.png`` in the job directory. :label:`overview`
+
+The full source code of all examples in this paper can be found online [#]_.
+
+.. [#] https://github.com/glotzerlab/signac-examples
+
+Research projects often address problems where questions change rapidly, data models are always in flux, and compute infrastructure varies widely from project to project.
+The **signac** data management framework is a tool designed by researchers, for researchers, to simplify the process of prototyping and performing reproducible scientific computations.
+It forgoes serializing complex data files into a database in favor of working with these files directly, providing fast indexing utilities for a set of directories.
+Using **signac**, a data space on the file system can be initialized, searched, and modified using either a Python or command-line interface.
+The companion package **signac-flow** interacts with the data space to generate and analyze data through reproducible workflows that scale from laptops to supercomputers.
+Arbitrary shell commands can be run by **signac-flow** as part of a workflow, making it as flexible as a script in any language of choice.
+
+With **signac**, file-based data and metadata are organized in folders and JSON files, respectively (see Figure :ref:`overview`).
+A **signac** data space, or "workspace," is composed of jobs, which are individual directories associated with a single primary key known as a "state point" stored in a file ``signac_statepoint.json`` in that directory.
```

Comment:
```suggestion
A **signac** data space, or "workspace", is composed of jobs, which are individual directories associated with a single primary key known as a "state point" stored in a file ``signac_statepoint.json`` in that directory.
```

Reply:
I was reading more about this. It seems like a convention, so I'll resolve the conversation.